test/e2e/scenarios/Snapshot/snapshot_update_with_tags.yaml

```diff
@@ -4,33 +4,15 @@ description: "In this test we create snapshot and update tags"
 #  - slow
 #  - blocked
 steps:
-  - id: "CLUSTER_INITIAL_CREATE"
-    description: "Create cluster"
-    create:
-      apiVersion: $CRD_GROUP/$CRD_VERSION
-      kind: Cluster
-      metadata:
-        name: cluster$RANDOM_SUFFIX
-      spec:
-        name: cluster$RANDOM_SUFFIX
-        nodeType: db.t4g.small
-        aclName: open-access
-        numShards: 1
-    wait:
-      status:
-        conditions:
-          ACK.ResourceSynced:
-            status: "True"
-            timeout: 7200
-  - id: "SNAPSHOT_INITIAL_CREATE"
-    description: "Create snapshot with no tags"
+  - id: "CREATE_INITIAL_SNAPSHOT"
+    description: "Create Initial Snapshot"
     create:
       apiVersion: $CRD_GROUP/$CRD_VERSION
       kind: Snapshot
       metadata:
         name: snapshot$RANDOM_SUFFIX
       spec:
-        clusterName: cluster$RANDOM_SUFFIX
+        clusterName: $SNAPSHOT_CLUSTER_NAME3
```

Comment:
I do not think we need to do this in bootstrap resource this should not cause flakiness.

Reply:
@nmvk I can change it back, but may I know why we don't use Cluster(Bootstrappable) if it is how we create cluster for Ack snapshot resource testing?

Reply:
Either use bootstrap resource or create a new cluster does the same thing. I thought it is better to use bootstrap resource because all other snapshot tests use this way.

Reply:
Btw, I will change it back and create a new PR for it (or merge it into other snapshot tests).